### PR TITLE
Logging update

### PR DESCRIPTION
**Pull request by krezefal**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/69 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by AronAtDell — APPROVED** [review]
Looks good to me. Thanks!

File: cmd/replication-controller/main.go [hunk 1]
```diff
@@ -193,13 +193,13 @@ func main() {
...
 	if err != nil {
-		setupLog.Error(err, "unable to start manager")
+		setupLog.Error(err, "Unable to start manager")
 		os.Exit(1)
 	}
 
 	controllerMgr, err := createControllerManager(ctx, mgr)
 	if err != nil {
-		setupLog.Error(err, "failed to configure the controller manager")
+		setupLog.Error(err, "Filed to configure the controller manager")
```
Comment (gallacher) [inline cmd/replication-controller/main.go:202]: Typo - change to "Failed"

File: controllers/csi-replicator/replicator_common.go [hunk 2]
```diff
@@ -35,12 +35,12 @@ func shouldContinue(class *storageV1.StorageClass, log logr.Logger, driverName s
...
 	// has a replica created for it
 	if value, ok := class.Parameters[controller.StorageClassReplicationParam]; !ok || value != controller.StorageClassReplicationParamEnabledValue {
-		log.V(common.InfoLevel).Info("StorageClass used to provision the PVC is not replication-enabled")
+		log.V(common.InfoLevel).Info("StorageClass used to provision the PVC is not replication-enabled", "StorageClass", class)
 		return false
 	}
 
 	// TODO: This should be removed when we start supporting RGs for SRDF Metro too
-	// Check for PowerMax SRDF Metro
+	// Check for PowerMax SRDF Metrot
```
Comment (gallacher) [inline controllers/csi-replicator/replicator_common.go:43]: Is this a typo?

**Review by gallacher — CHANGES_REQUESTED** [review]
Minor typos need fixing

File: controllers/csi-replicator/replicator_common.go (hunk 2 above)
Comment (krezefal) [reply controllers/csi-replicator/replicator_common.go:43]: yes of course

**Review by walker2 — APPROVED** [review]

**Review by maxan98 — APPROVED** [review]

**Review by AronAtDell — DISMISSED** [review]
@walker2, @krezefal , @maxan98 - Feel free to merge up. Thanks!

File: controllers/csi-replicator/dellcsireplicationgroup_controller.go [hunk 3]
```diff
@@ -131,29 +134,40 @@ func updateRGSpecWithActionResult(rg *storagev1alpha1.DellCSIReplicationGroup, r
...
 
-func getActionResultFromActionAnnotation(actionAnnotation ActionAnnotation) (*ActionResult, error) {
+func getActionResultFromActionAnnotation(ctx context.Context, actionAnnotation ActionAnnotation) (*ActionResult, error) {
+	log := common.GetLoggerFromContext(ctx)
+	log.V(common.InfoLevel).Info("Getting result from action annotation..")
+
 	var finalErr error
 	finalError := false
 	if actionAnnotation.FinalError != "" {
-		finalErr = fmt.Errorf(actionAnnotation.FinalError)
```
Comment (walker2) [inline controllers/csi-replicator/dellcsireplicationgroup_controller.go:143]: This shouldn't be removed, it breaks everything

**Review by walker2 — APPROVED** [review]

**Review by maxan98 — APPROVED** [review]

**Pull request merged**